pkgs/by-name/co/cosmic-settings-daemon/package.nix

```diff
@@ -0,0 +1,34 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+, pkg-config
+, udev
+
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "cosmic-settings-daemon";
+  version = "unstable-2023-12-11";
+
+  src = fetchFromGitHub {
+    owner = "pop-os";
+    repo = pname;
+    rev = "29b47fed5f64e1f2ee3494d9230256c9fb1873aa";
+    hash = "sha256-+AAy8iVYCCJUVsA5oDRsWu9tYF6x1m4yiebw5CTz4i0=";
+  };
+
+  cargoLock = {
+    lockFile = ./Cargo.lock;
+  };
```

Comment:
I don't think we need the lockfile here and can just use the hash.

Reply:
Same reasoning as the other review comment I left (git dependencies are not supported by rustPlatform at this time)

Reply:
but there are also none used here

Reply:
Fixed & changed